docs/general/roles.md

```diff
@@ -19,13 +17,13 @@ There are three System Roles:
 
 The Super Admin can access all user features.
 
-| Function | Role | View | Create, Edit, Assign, Unassign | Delete | BaseURLs |
-| --- | --- | --- | --- | --- | --- |
-| MANAGE OWN PROFILE | Super Admin | YES | YES | YES | /profile
-| MANAGE OWN EVENTS | Super Admin | YES | YES | YES | /create, /events, /events/ID
-| IMPORT/EXPORT OWN EVENTS | Super Admin | YES | YES | YES |
-| MANAGE OWN SESSIONS | Super Admin | YES | YES | YES |
-| MANAGE SESSIONS OF OWN EVENTS | Super Admin | YES | YES | YES |
+Function | Role | View | Create, Edit, Assign, Unassign | Delete | BaseURLs
```

Comment:
instead add `|` at the end of those missing and check for those rows with less than 6 columns